facebookresearch_semi-supervised-ImageNet1K-models_resnext.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+title: Semi-supervised and semi-weakly supervised ImageNet Models
+summary: ResNet and ResNext models introduced in the "Billion scale semi-supervised learning for image classification" paper
+category: researchers
+image: ssl-image.png
+author: Facebook AI
+tags: [vision]
+github-link: https://github.com/facebookresearch/semi-supervised-ImageNet1K-models/blob/master/hubconf.py
+featured_image_1: ssl-image.png
+featured_image_2: no-image
+accelerator: cuda-optional
+order: 10
+---
+
+```python
+import torch
+
+# === SEMI-WEAKLY SUPERVISED MODELSP RETRAINED WITH 940 HASHTAGGED PUBLIC CONTENT === 
+model = torch.hub.load('facebookresearch/semi-supervised-ImageNet1K-models', 'resnext18_swsl')
```

Comment:
`resnext18_swsl` is not available in https://github.com/facebookresearch/semi-supervised-ImageNet1K-models/blob/master/hubconf.py. Looks like a typo?